extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityTodoListModel.class.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class CommunityPageSpecialCommunityTodoListModel {
+	public function getData() {
+		$title = Title::newFromText(
+			wfMessage( 'communitypage-todo-module-page-name' )->inContentLanguage()->plain(),
```

Comment:
@Grunny Your suggested alternate code works well, but I am now missing the edit link which was:

`'editUrl' => $title->getFullURL( ['action' => 'edit'] ),`

I tried `Title::newFromText(` $message->plain(), NS_MEDIAWIKI );` to get the Title, but it just gives null.

Any suggestions on how to get the proper edit url in the context of your suggested code?